### PR TITLE
Add text for prices and fix transfer button

### LINKED ISSUE #944: Line for adding group badges should include the current price
Mostly so I don't have to remember the rules.  Because depending on THAT is not the smartest plan.

### PR DESCRIPTION
**Pull request by kitsuta**
Fixes #944 #954 and #946.

### CODE REVIEW

File: uber/models.py [hunk 1]
```diff
@@ -603,7 +607,7 @@ def badge(self):
         return badge
 
     @property
```
Comment (EliAndrewC) [inline uber/models.py]: I could have sworn we already fixed this typo.

File: uber/templates/preregistration/confirm.html [hunk 2]
```diff
@@ -58,12 +58,14 @@
     <td></td>
     <td> <input type="submit" value="{% if attendee.placeholder %}Register{% else %}Update My Info{% endif %}" id="updateButton"/> </td>
 </tr>
```
Comment (EliAndrewC) [inline uber/templates/preregistration/confirm.html]: I swear we already had this if statement in place.  Must have gotten deleted somehow.

Comment (kitsuta) [inline uber/templates/preregistration/confirm.html]: Actually, I fixed it in ac-staging.

Comment (EliAndrewC) [inline uber/templates/preregistration/confirm.html]: Ah, I must have been thinking of that PR.

**Pull request merged**